sort/prioritized.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gxsort
+
+// Prioritized is the abstraction of priority.
+type Prioritized interface {
```

Comment:
what's this interface use for? add some unit test for it.  And how about rename to `Prioritizer`

Reply:
Used to declare some thing that should be ordered by its priority.
For example, the listener in dubbo2.7.5 should be ordered by priority. 
It's just a interface, we don't need UT.